crit-go/cmd/root.go

```diff
@@ -0,0 +1,38 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+	"os"
+)
+
+var inloc, outloc, dir, what, cfgFile string
+var pretty, nopl bool
+
+// rootCmd represents the base command when called without any subcommands
+
+var rootCmd = &cobra.Command{
+	Use:   "crit-go",
+	Short: "CRIT is a feature-rich replacement for existing criu show",
+	Long: `CRIT is a feature-rich replacement for existing "criu show". 
+	This version is written in Go for usage with Go codebase, crit is also available in python
+	for more information visit https://criu.org/CRIT `,
+	Run: func(cmd *cobra.Command, args []string) {
+		fmt.Println("crit executed")
+	},
+}
+
+func Execute() {
+	if err := rootCmd.Execute(); err != nil {
+		fmt.Println(err)
```

Comment:
stderr

Reply:
what i will do is simply fix the cli on this branch itself will be easy to keep track of things now that the pb files which made up the majority of this branch are removed.

Reply:
Resolved with logging